test/integration_test.js

```diff
@@ -17,34 +17,26 @@ const properties = {
 };
 let clientService;
+let cassandraClient;
 
 describe('Integration test on ClientService', async () => {
-  const mockedFunctionId = 'TestFunction';
+  const mockedByteContract = fs.readFileSync(
+      __dirname + '/StateUpdater.class');
+  const mockedByteFunction = fs.readFileSync(
+      __dirname + '/TestFunction.class');
+  const mockedFunctionId = `TestFunctions${Date.now()}`;
   const mockedContractId = `StateUpdater${Date.now()}`;
   const mockedContractName = 'com.org1.contract.StateUpdater';
   const mockedFunctionName = 'com.org1.function.TestFunction';
-  const mockedAssetId = 'mockedAssetId';
+  const mockedAssetId = `mockedAssetId${Date.now()}`;
   const mockedState = 1;
   const mockedContractArgument = {
     asset_id: mockedAssetId,
     state: mockedState,
   };
-  const mockedFunctionArgument = {
-    asset_id: mockedAssetId,
-    state: mockedState,
-    _function_: mockedFunctionId,
-  };
-  const property = {
+  const contractProperty = {
     properties: 'bar',
   };
-  const functionArgument = {
-    asset_id: mockedAssetId,
-    state: mockedState,
-  };
-  const mockedByteContract = fs.readFileSync(
-      __dirname + '/StateUpdater.class');
-  const mockedByteFunction = fs.readFileSync(
-      __dirname + '/TestFunction.class');
   clientService = new ClientService(properties);
   describe('registerCertificate', () => {
     it('should works as expected', async () => {
```

Comment:
First of all, 
```suggestion
    it('should work as expected', async () => {
```
but, it is too general not informative because all the tests are checking if they are working as expected.
So let's make it clear. For example, `registerCertificate should return 200 when correct inputs are specified.`.